.dockerignore

```diff
@@ -1,5 +1,16 @@
 *
-!gitea
```

Comment:
Heck, all there things you've added in here should still go into the first build...